english/the-nights/the-nights.md

```diff
@@ -1,5 +1,5 @@
 One day you'll leave this world behind
-So live a life you will remember.
+So [live a life](../life_well_lived/purpose.md) you will remember.
  [Go Back](../marshmallow.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.